src/Symfony/Bundle/FrameworkBundle/Resources/config/fragment_renderer.xml

```diff
@@ -11,7 +11,7 @@
 
     <services>
         <service id="fragment.handler" class="Symfony\Component\HttpKernel\DependencyInjection\LazyLoadingFragmentHandler">
-            <argument type="service" id="service_container" />
+            <argument /> <!-- fragment renderer locator -->
```

Comment:
are you talking about `getSession()` in the session listener instead? If no, could you elaborate?